run_dir/static/js/instrument_logs.js

```diff
@@ -0,0 +1,54 @@
+
+function init_listjs() {
+    // Setup - add a text input to each footer cell
+    $('#instrument_logs_table tfoot th').each( function () {
+      var title = $('#instrument_logs_table thead th').eq( $(this).index() ).text();
+      $(this).html( '<input size=10 type="text" placeholder="Search '+title+'" />' );
+    } );
+                             
+    var table = $('#instrument_logs_table').DataTable({
+      "paging":false,
+      "info":false,
+      "order": [[ 0, "desc" ]]
+    });
+
+    //Add the bootstrap classes to the search thingy
```

Comment:
Okay, so it's for the whole table search thingy. but since there's a .remove, it's not there. I think I'll keep it.